src/WCSimConstructMaterials.cc

```diff
@@ -1551,7 +1551,11 @@ void WCSimDetectorConstruction::ConstructMaterials()
   // OpWaterWLSSurface->SetSigmaAlpha(0.1); // TODO: What's this?
 
   // MATERIAL properties
+#if 0
```

Comment:
I don't think this is a good implementation. Making a configurable command like those in `macros/tuning_parameters.mac` is better in my opinion.

Reply:
well, the thing i am submitting does work. I don't know how to make it better. I have tried, but couldn't. Although the eljen type was there before, it was not assigned via tuning parameters. So I am really just replicating what was there before.
Can you suggest how to actually change it? Or else please just approve it.

Reply:
yes I tried but I couldn't do it. Can you actually try to implement it?
It should have been done even before for the eljen class, so I don't really think it's my job to fix it